org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
well im not going to die for my opinion but we definitely should do
somethhng with test utils. writing test now is let me say
discouraging. let talk on monday

2015-05-15 16:24 GMT+02:00, Simon Schäfer <notifications@github.com>:
>>   *
>>   */
>> -object SDTTestUtils extends HasLogger {
>> +trait SDTTestUtils extends HasLogger {
>
> Full test frameworks have some other design goals to their code than what we
> have. Changing it to a trait means that it needs more Scaladoc because it
> needs to explain why and where it should instantiated.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/scala-ide/scala-ide/pull/948/files#r30412208